imports/plugins/core/orders/client/templates/workflow/shippingSummary.js

```diff
@@ -34,10 +34,14 @@ Template.coreOrderShippingSummary.onRendered(function () {
 
   if (order.workflow) {
     if (order.workflow.status === "coreOrderCreated") {
-      order.workflow.status = "coreOrderCreated";
       Meteor.call("workflow/pushOrderWorkflow", "coreOrderWorkflow", "coreOrderCreated", order);
     }
   }
+  if (order.workflow) {
```

Comment:
You can combine the `if (order.workflow) {` block with the one above. You can also eliminate one of those the way we just did